src/navigation/orbitalnavigator.cpp

```diff
@@ -698,6 +718,24 @@ OrbitalNavigator::OrbitalNavigator()
             LINFO("Camera roll has been enabled");
         }
     });
+
+    addProperty(_shouldRotateAroundUp);
+    _upToUseForRotation.addOptions({
+        {
+            static_cast<int>(UpDirectionChoice::XAxis),
+            "Local X"
+        },
+        {
+            static_cast<int>(UpDirectionChoice::YAxis),
+            "Local Y"
+        },
+        {
+            static_cast<int>(UpDirectionChoice::ZAxis),
+            "Local Z"
+        },
```

Comment:
```suggestion
        { static_cast<int>(UpDirectionChoice::XAxis), "Local X" },
        { static_cast<int>(UpDirectionChoice::YAxis), "Local Y" },
        { static_cast<int>(UpDirectionChoice::ZAxis), "Local Z" }
```

is a bit more compact to me and seems readable to me